docs/technical_reference/unit_models/stoichiometric_reactor.rst

```diff
@@ -0,0 +1,205 @@
+               "dissolution_stoichiometric": {"Ca_2+": 1, "H2O": 1},
+         },
+      }
+
+Example dictionary for defined precipitant that would form during reaction (Calcite - CaCO3, and Brucite - MgOH)
+
+
+.. code-block::
+
+   precipitants = {
+      "Calcite": {
+            "mw": 100.09 * pyunits.g / pyunits.mol,
+            "precipitation_stoichiometric": {"Ca_2+": 1, "HCO3_-": 1},
+      },
+      "Brucite": {
+            "mw": 58.3197 * pyunits.g / pyunits.mol,
+            "precipitation_stoichiometric": {"Mg_2+": 1, "H2O": 1},
+      },
+   }
+
+The user can then provide the model with specified dictionary to produce a stoichiometric reactor that only perform dissolution, precipitation, or both. 
+
+.. code-block::
+
+   # unit for only adding a reagent
+   m.fs.chemical_addition = StoichiometricReactor(property_package=m.fs.properties,  reagent=reagents)
+   # The user must the specify how much reagent to add
+   m.fs.chemical_addition.reagent_dose["Na2CO3"].fix(1e-3)
+   m.fs.chemical_addition.reagent_dose["CaO"].fix(1e-3)
+   
+   # unit for only precipitating specified species out of the feed
+   m.fs.precipitation_reactor = StoichiometricReactor(property_package=m.fs.properties, precipitants=precipitants)
+   # The user must then specify how much precipitant to form 
+   m.fs.precipitation_reactor.flow_mass_precipitate["Calcite"].fix(1e-3)
+   m.fs.precipitation_reactor.flow_mass_precipitate["Brucite"].fix(1e-4)
+   # The user must also specify solids fraction in waste stream
+   m.fs.unit.waste_mass_frac_precipitate.fix(0.2)
+   
+   # unit for addition and precipitation (e.g. traditional Lime/Soda ash softening process)
+   m.fs.lime_soda_softening = StoichiometricReactor(property_package=m.fs.properties, reagent=reagents, precipitants=precipitants)
+   # The user must the specify how much reagent to add and precipitant to form
+   m.fs.lime_soda_softening.reagent_dose["Na2CO3"].fix(1e-3)
+   m.fs.lime_soda_softening.reagent_dose["CaO"].fix(1e-3)
+   m.fs.lime_soda_softening.flow_mass_precipitate["Calcite"].fix(1e-3)
+   m.fs.lime_soda_softening.flow_mass_precipitate["Brucite"].fix(1e-4)
+   # The user must also specify solids fraction in waste stream
+   m.fs.lime_soda_softening.waste_mass_frac_precipitate.fix(0.2)
+
+Sets
+----
+.. csv-table::
+   :header: "Description", "Symbol", "Indices"
+
+   "time", ":math:`t`", "[0]"
+   "phases", ":math:`p`", "['Liq']"
+   "components", ":math:`j`", "['H2O', solutes]"
+   "reagents", ":math:`\text{reagents}`",[reagent]
+   "precipitants", ":math:`\text{precipitants}`",[precipitants]
+
+Variables
+----------
+
+.. csv-table::
+   :header: "Description", "Variable Name", "Index", "Units"
+   
+   "Reagent dose", reagent_dose,[reagent],kg/:math:`\text{m}^3`
+   "Reagent density", density_reagent,[reagent],kg/:math:`\text{m}^3`
+   "Reagent flow mass", flow_mass_reagent,[reagent],kg/s
+   "Reagent flow volume", flow_vol_reagent,[reagent],:math:`\text{m}^3`/s
```

Comment:
m3 formatting is off - this should work: `:math:`kg/m^3``

Reply:
What do other docs use, I copied this from ED docs =P